jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/ExtractTarStep.java

```diff
@@ -123,7 +124,15 @@ public LocalImage call()
       Path file = destination.resolve(layerFiles.get(index));
 
       // Compress layers if necessary and calculate the digest/size
-      Blob blob = layersAreCompressed ? Blobs.from(file) : Blobs.compress(Blobs.from(file));
+      Blob blob = Blobs.from(file);
```

Comment:
should we solve the underlying issue or calling `Blobs.writeTo` should *not* close the output stream according to the javadoc on the interface https://github.com/GoogleContainerTools/jib/blob/1dd28e1224158bf9cb9d01e6a5edb6423c2d8bbf/jib-core/src/main/java/com/google/cloud/tools/jib/blob/Blob.java#L26

but our FileBlob implementation appears to be doing exactly that?

Ofcourse this can be in a followup PR.

Reply:
I'll just file an issue for this for now. I don't *think* there's a problem with closing the output stream, but we should be clear about whether or not the stream is closed in each implementation.